lang/hr/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Semantičko verzioniranje 2.0.0
+language: hr
+---
+
+Semantičko verzioniranje 2.0.0
+==============================
+
+Sažetak
+-------
+
+Uzevši oznake verzije VEĆA.MANJA.ZAKRPA, povećavajte:
+
+1. VEĆU verziju kada napravite inkompatibilne promjene u APIju,
+1. MANJU verziju kada dodate unatrag-kompatibilne funkcionalnosti i
+1. ZAKRPA verziju kada napravite unatrag-kompatibilne ispravke bugova.
+
+Dodatne oznake za pred-izdanja i metapodatke builda mogu se koristiti kao proširenje na VEĆA.MANJA.ZAKRPA format.
+
+Uvod
+------------
+
+U svijetu upravljanja softverom postoji strašno mjesto koje zovemo
+"pakao ovisnosti." Što više vaš sustav raste i što više paketa
+integrirate u vaš softver, vjerojatnije je da ćete se, jednoga dana,
+naći u toj jami očaja.
```

Comment:
A nitpick, maybe "u tom tamnom kutku" is more in the spirit of Croatian. This roughly translates to "in this dark corner".

Reply:
i don't think "in this dark corner" is nearly as *dramatic* as intended :)